utils/na_tag_rpc.diff

```diff
@@ -0,0 +1,39 @@
+From a9b9ce490ebd17fb31374529b44bc42222d4db3a Mon Sep 17 00:00:00 2001
+From: Jinshan Xiong <jinshanx@google.com>
+Date: Thu, 13 Feb 2025 13:46:25 -0800
+Subject: [PATCH] NA tcp: Add FI_TAG_RPC support
```

Comment:
this should be .patch rather than .diff

We currently ignore *.patch for spellcheck (not an issue here) and we should be consistent.